### PR TITLE
chore(license-book): update license book for 7.21.0

### PR DESCRIPTION
**Pull request by tasso94**
related to camunda/camunda-bpm-platform#3830

### CODE REVIEW

**Review by yanavasileva — COMMENTED** [review]
👍 Looks good.

📓 Some `href` are missing or broken in this version. If I understood correctly they will be fix after merge and running the CI.

**Comment by tasso94** [comment]
Can you point me to the places where the `href` is broken?

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 1]
```diff
@@ -988,7 +1261,50 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
...
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+</pre></details></div>
+<div><details><summary><a href="git+https://github.com/bpmn-io/feel-editor.git">@bpmn-io/feel-editor@1.2.0</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:1307]: URL broken but was broken before as well.

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 2]
```diff
@@ -1009,7 +1325,7 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.</pre></details></div>
-<div><details><summary><a href="git+https://github.com/bpmn-io/feel-linter.git">@bpmn-io/feel-lint@0.1.1</a> (MIT)</summary><pre>The MIT License (MIT)
+<div><details><summary><a href="git+https://github.com/bpmn-io/feel-linter.git">@bpmn-io/feel-lint@1.2.0</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:1328]: URL broken but was broken before as well.

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 3]
```diff
@@ -1956,7 +2272,7 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
 </pre></details></div>
-<div><details><summary><a href="https://github.com/sindresorhus/array-move">array-move@3.0.1</a> (MIT)</summary><pre>MIT License
+<div><details><summary><a href="sindresorhus/array-move">array-move@3.0.1</a> (MIT)</summary><pre>MIT License
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:2275]:
```suggestion
<div><details><summary><a href="https://github.com/sindresorhus/array-move">array-move@3.0.1</a> (MIT)</summary><pre>MIT License
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 4]
```diff
@@ -3778,7 +4581,7 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.</pre></details></div>
-<div><details><summary><a href="https://github.com/developit/mitt">mitt@3.0.0</a> (MIT)</summary><pre>MIT License
+<div><details><summary><a href="developit/mitt">mitt@3.0.1</a> (MIT)</summary><pre>MIT License
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:4584]:
```suggestion
<div><details><summary><a href="https://github.com/developit/mitt">mitt@3.0.1</a> (MIT)</summary><pre>MIT License
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 5]
```diff
@@ -6030,7 +7016,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/jshttp/cookie">cookie@0.5.0</a> (MIT)</summary><pre>(The MIT License)
+<div><details><summary><a href="jshttp/cookie">cookie@0.6.0</a> (MIT)</summary><pre>(The MIT License)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:7019]:
```suggestion
<div><details><summary><a href="https://github.com/jshttp/cookie">cookie@0.6.0</a> (MIT)</summary><pre>(The MIT License)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 6]
```diff
@@ -6871,7 +8038,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/jonschlinkert/is-plain-object">is-plain-object@5.0.0</a> (MIT)</summary><pre>The MIT License (MIT)
+<div><details><summary><a href="jonschlinkert/is-plain-object">is-plain-object@5.0.0</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8041]:
```suggestion
<div><details><summary><a href="https://github.com/jonschlinkert/is-plain-object">is-plain-object@5.0.0</a> (MIT)</summary><pre>The MIT License (MIT)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 7]
```diff
@@ -6912,7 +8079,7 @@ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
 IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE
 </pre></details></div>
-<div><details><summary><a href="https://github.com/nodeca/js-yaml">js-yaml@4.1.0</a> (MIT)</summary><pre>(The MIT License)
+<div><details><summary><a href="nodeca/js-yaml">js-yaml@4.1.0</a> (MIT)</summary><pre>(The MIT License)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8082]:
```suggestion
<div><details><summary><a href="https://github.com/nodeca/js-yaml">js-yaml@4.1.0</a> (MIT)</summary><pre>(The MIT License)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 8]
```diff
@@ -6934,7 +8101,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/lodash/lodash">lodash@4.17.21</a> (MIT)</summary><pre>Copyright OpenJS Foundation and other contributors <https://openjsf.org/>
+<div><details><summary><a href="lodash/lodash">lodash@4.17.21</a> (MIT)</summary><pre>Copyright OpenJS Foundation and other contributors <https://openjsf.org/>
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8104]:
```suggestion
<div><details><summary><a href="https://github.com/lodash/lodash">lodash@4.17.21</a> (MIT)</summary><pre>Copyright OpenJS Foundation and other contributors <https://openjsf.org/>
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 9]
```diff
@@ -6982,7 +8149,7 @@ maintained libraries used by this software which have their own
 licenses; we recommend you read them, as their terms may differ from the
 terms above.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/lodash/lodash">lodash.debounce@4.0.8</a> (MIT)</summary><pre>Copyright jQuery Foundation and other contributors <https://jquery.org/>
+<div><details><summary><a href="lodash/lodash">lodash.debounce@4.0.8</a> (MIT)</summary><pre>Copyright jQuery Foundation and other contributors <https://jquery.org/>
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8152]:
```suggestion
<div><details><summary><a href="https://github.com/lodash/lodash">lodash.debounce@4.0.8</a> (MIT)</summary><pre>Copyright jQuery Foundation and other contributors <https://jquery.org/>
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 10]
```diff
@@ -7030,7 +8197,7 @@ maintained libraries used by this software which have their own
 licenses; we recommend you read them, as their terms may differ from the
 terms above.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/wooorm/lowlight">lowlight@1.20.0</a> (MIT)</summary><pre>(The MIT License)
+<div><details><summary><a href="wooorm/lowlight">lowlight@1.20.0</a> (MIT)</summary><pre>(The MIT License)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8200]:
```suggestion
<div><details><summary><a href="https://github.com/wooorm/lowlight">lowlight@1.20.0</a> (MIT)</summary><pre>(The MIT License)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 11]
```diff
@@ -7588,7 +8690,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/jonschlinkert/repeat-string">repeat-string@1.6.1</a> (MIT)</summary><pre>The MIT License (MIT)
+<div><details><summary><a href="jonschlinkert/repeat-string">repeat-string@1.6.1</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8693]:
```suggestion
<div><details><summary><a href="https://github.com/jonschlinkert/repeat-string">repeat-string@1.6.1</a> (MIT)</summary><pre>The MIT License (MIT)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 12]
```diff
@@ -7699,7 +8801,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/sindresorhus/serialize-error">serialize-error@8.1.0</a> (MIT)</summary><pre>MIT License
+<div><details><summary><a href="sindresorhus/serialize-error">serialize-error@8.1.0</a> (MIT)</summary><pre>MIT License
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8804]:
```suggestion
<div><details><summary><a href="https://github.com/sindresorhus/serialize-error">serialize-error@8.1.0</a> (MIT)</summary><pre>MIT License
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 13]
```diff
@@ -7743,7 +8867,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/stampit-org/stampit">stampit@4.3.2</a> (MIT)</summary><pre>The MIT License (MIT)
+<div><details><summary><a href="git@github.com:stampit-org/stampit.git">stampit@4.3.2</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8870]:
```suggestion
<div><details><summary><a href="https://github.com/stampit-org/stampit.git">stampit@4.3.2</a> (MIT)</summary><pre>The MIT License (MIT)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 14]
```diff
@@ -7765,7 +8889,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/swagger-api/swagger-js">swagger-client@3.19.10</a> (Apache-2.0)</summary><pre>
+<div><details><summary><a href="git@github.com:swagger-api/swagger-js.git">swagger-client@3.25.0</a> (Apache-2.0)</summary><pre>
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:8892]:
```suggestion
<div><details><summary><a href="https://github.com/swagger-api/swagger-js.git">swagger-client@3.25.0</a> (Apache-2.0)</summary><pre>
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 15]
```diff
@@ -7968,7 +9092,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/swagger-api/swagger-ui">swagger-ui@5.1.0</a> (Apache-2.0)</summary><pre>
+<div><details><summary><a href="git@github.com:swagger-api/swagger-ui.git">swagger-ui@5.11.2</a> (Apache-2.0)</summary><pre>
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:9095]:
```suggestion
<div><details><summary><a href="https://github.com/swagger-api/swagger-ui.git">swagger-ui@5.11.2</a> (Apache-2.0)</summary><pre>
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 16]
```diff
@@ -2295,7 +2590,7 @@ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
 IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/JedWatson/classnames.git">classnames@2.3.2</a> (MIT)</summary><pre>The MIT License (MIT)
+<div><details><summary><a href="git+https://github.com/JedWatson/classnames.git">classnames@2.5.1</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:2593]:
```suggestion
<div><details><summary><a href="https://github.com/JedWatson/classnames.git">classnames@2.5.1</a> (MIT)</summary><pre>The MIT License (MIT)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 17]
```diff
@@ -2317,7 +2612,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/zenorocha/clipboard.js">clipboard@2.0.11</a> (MIT)</summary><pre>MIT License
+<div><details><summary><a href="zenorocha/clipboard.js">clipboard@2.0.11</a> (MIT)</summary><pre>MIT License
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:2615]:
```suggestion
<div><details><summary><a href="https://github.com/zenorocha/clipboard.js">clipboard@2.0.11</a> (MIT)</summary><pre>MIT License
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 18]
```diff
@@ -2339,7 +2634,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/lukeed/clsx">clsx@1.2.1</a> (MIT)</summary><pre>MIT License
+<div><details><summary><a href="lukeed/clsx">clsx@2.1.0</a> (MIT)</summary><pre>MIT License
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:2637]:
```suggestion
<div><details><summary><a href="https://github.com/lukeed/clsx">clsx@2.1.0</a> (MIT)</summary><pre>MIT License
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 19]
```diff
@@ -2607,7 +2840,7 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.</pre></details></div>
-<div><details><summary><a href="https://github.com/bpmn-io/diagram-js-grid">diagram-js-grid@0.2.0</a> (MIT)</summary><pre>The MIT License (MIT)
+<div><details><summary><a href="git@github.com:bpmn-io/diagram-js-grid.git">diagram-js-grid@0.2.0</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:2843]:
```suggestion
<div><details><summary><a href="https://github.com/bpmn-io/diagram-js-grid.git">diagram-js-grid@0.2.0</a> (MIT)</summary><pre>The MIT License (MIT)
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 20]
```diff
@@ -3330,7 +4133,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/developit/htm">htm@3.1.1</a> (Apache-2.0)</summary><pre>
+<div><details><summary><a href="developit/htm">htm@3.1.1</a> (Apache-2.0)</summary><pre>
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:4136]:
```suggestion
<div><details><summary><a href="https://github.com/developit/htm">htm@3.1.1</a> (Apache-2.0)</summary><pre>
```

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 21]
```diff
@@ -4123,7 +4926,7 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.</pre></details></div>
-<div><details><summary><a href="https://github.com/preactjs/preact">preact@10.13.2</a> (MIT)</summary><pre>The MIT License (MIT)
+<div><details><summary><a href="preactjs/preact">preact@10.19.3</a> (MIT)</summary><pre>The MIT License (MIT)
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:4929]:
```suggestion
<div><details><summary><a href="https://github.com/preactjs/preact">preact@10.19.3</a> (MIT)</summary><pre>The MIT License (MIT)
```

**Review by yanavasileva — COMMENTED** [review]
I added suggestions where I think CI will fail as it will attempt to open a relative path.
There are more links that won't open existing page but at least the CI won't fail for them (for example search for `href="g`).

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md (hunk 5 above)
Comment (yanavasileva) [reply content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:7019]: @tasso94 that url throws 404 without my suggestion.

**Review by yanavasileva — COMMENTED** [review]
👍 Great work. Please check the only non-resolved comment for `cookie@0.6.0`, docs CI will fail because of it.

**Pull request merged**